include/pros/llemu.h

```diff
@@ -52,6 +52,16 @@ typedef struct lcd_s {
 	                               // multitouch support)
 } lcd_s_t;
 
+typedef enum lcd_text_align_e {
+	E_LCD_TEXT_ALIGN_LEFT = 0,
+	E_LCD_TEXT_ALIGN_CENTER = 1,
+	E_LCD_TEXT_ALIGN_RIGHT = 2
+} lcd_text_align_e_t;
+
+#define ALIGN_LEFT pros::E_LCD_TEXT_ALIGN_LEFT 
+#define ALIGN_CENTER pros::E_LCD_TEXT_ALIGN_CENTER
+#define ALIGN_RIGHT pros::E_LCD_TEXT_ALIGN_RIGHT
```

Comment:
these definitions of these three macros is dependent on whether the user is in a C++ environment or not.

the simplest solution would be to refactor them in-place like:
```cpp
#ifdef __cplusplus
#define ALIGN_LEFT pros::E_LCD_TEXT_ALIGN_LEFT
#define ALIGN_CENTER pros::E_LCT_TEXT_ALIGN_CENTER
#define ALIGN_RIGHT pros::E_LCD_TEXT_ALIGN_RIGHT
#else
#define ALIGN_LEFT E_LCD_TEXT_ALIGN_LEFT
#define ALIGN_CENTER E_LCT_TEXT_ALIGN_CENTER
#define ALIGN_RIGHT E_LCD_TEXT_ALIGN_RIGHT
#endif
```

Reply:
I agree